src/core/store/modules/tokens/tokens.selectors.ts

```diff
@@ -106,6 +87,7 @@ export function createToken(props: CreateTokenProps): TokenView {
     description: tokenData?.metadata?.description ?? '',
     website: tokenData?.metadata?.website ?? '',
     isZapable: tokenData?.supported.zapper ?? false,
+    sourceIsZapper: tokenData?.dataSource === 'zapper',
```

Comment:
As mentioned in https://github.com/yearn/yearn-finance-v3/pull/674#discussion_r875331720 could we just have the source here? When we want to extend this to other data sources it would be better to have a more extendable data structure

Reply:
what I dont like about extending external types like `dataSource`, is that we couple our app to it, and if it gets changed for some reason (even though its our own SDK), it breaks all layers of the app. Here, we only need to change the adapter layer, the selector in this case.